### ISSUE TITLE
Feeds are not generated

### ISSUE THREAD

**Issue by TC01** [TC01/pelican-mboxreader#5]
To prevent feeds getting written twice and pelican complaining, I disabled generation of feeds. Or, more accurately, feeds aren't generated for the mbox categories; other feeds still get generated, of course.

This is not ideal; at the very least the category specific feeds should get created.

**Comment by TC01** [comment]
Hm, it seems that articles are now properly added to the global all feed, so we just need to implement a generate_feeds that generates feeds for our new categories.

**Comment by TC01** [comment]
To do this properly, we will need to be able to overwrite feeds, because category- and article- collisions are now supported.

**Comment by TC01** [comment]
So, just to update the world: I got a patch to add the ability to overwrite feeds merged into Pelican some time ago, so once 3.7 lands I think this can be done.

**Comment by TC01** [comment]
pelican 3.7 has been released so this *should* now be possible.
